### PR TITLE
Bitbucket api

### PR DESCRIPTION
**Pull request by ghost**
Helper to retrieve tasks from bitbucket

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -22,8 +22,11 @@ gem 'simple_form'
 gem 'thin'
 gem 'omniauth'
 gem 'omniauth-bitbucket'
+gem 'httparty'
```
Comment (jbrewton) [inline Gemfile]: :tada:

File: app/helpers/bitbucket_helper.rb [hunk 2]
```diff
@@ -1,15 +1,56 @@
...
+    "https://bitbucket.org/api/1.0/repositories/" 
+  end
+
+  def base_uri_v2
+    "https://bitbucket.org/api/2.0/repositories/"
+  end
+
+  def repo_exists
+    if current_user
+      url = base_uri_v1 + current_user.uid + '/tasklistapp'
```
Comment (jbrewton) [inline app/helpers/bitbucket_helper.rb:16]: Future proof this with URI joins one day but today is not that one day.

**Comment by jbrewton** [comment]
the lines are adequate, may they be submitted. :shipit:

**Comment by MKravchak** [comment]
:shell: :sake: :dancers: :camel: :last_quarter_moon: :package: :yen: :tanabata_tree:

**Pull request merged**